packs/newrelic/config.schema.yaml

```diff
@@ -0,0 +1,38 @@
+---
+  api_url:
+    description: "New Relic API URL. Default https://api.newrlic.com/v2/"
+    type: "string"
+    required: true
+    secret: false
+    default: "https://api.newrelic.com/v2/"
+  api_key:
+    description: "New Relic API Key"
+    type: "string"
+    required: true
+    secret: true
+  sensor_config:
+    description: "Sensor specific settings."
+    type: "object"
+    required: true
+    additionalProperties: false
+    properties:
+      host:
+        type: "string"
+        description: "IP Address sensor should listen on. Default 0.0.0.0"
+        required: false
+        default: "0.0.0.0"
+      port:
+        type: "integer"
+        description: "Port Address sensor should listen on. Default 10001"
+        required: false
+        default: 10001
```

Comment:
This reminds me - I actually need to check / verify that default values work with nested objects.